src/include/distributed/intermediate_result_pruning.h

```diff
@@ -17,7 +17,13 @@
  * UINT32_MAX is reserved in pg_dist_node, so we can use it safely.
  */
 #define LOCAL_NODE_ID UINT32_MAX
-#define LOCAL_HOST_NAME "localhost" /* connect to local backends using this name */
+
+/*
+ * If you want to connect to the current node use `LocalHostName`, which is a GUC, instead
+ * of the hardcoded loopback hostname. Only if you really need the loopback hostname use
+ * this define.
+ */
+#define LOCAL_HOST_NAME "localhost"
```

Comment:
This define is also used in enterprise, hence I put it back after removing it at first.
If you feel strong about it I will remove it, but it will cause some issues with the merge to enterprise :/